pkg/controller/certificates/sync.go

```diff
@@ -72,6 +78,11 @@ func (c *Controller) Sync(ctx context.Context, crt *v1alpha1.Certificate) (err e
 		return err
 	}
 
+	issuerType, err := issuer.NameForIssuer(issuerObj)
+	if err != nil {
+		return err
+	}
```

Comment:
This var isn't used until much later in the function - best to move it down to nearer to make it clear why it's being obtained.

Reply:
I had it further down initially but was advised to move it closer to the top because it's used in multiple place - will change

Reply:
As per other comments, this can go (can be computed by the metrics package)